elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -123,6 +123,7 @@ elytron.file-audit-log.remove=Remove the audit logger resource.
 #Attributes
 elytron.file-audit-log.path=Path of the file to be written.
 elytron.file-audit-log.relative-to=The relative path to the audit log.
+elytron.file-audit-log.autoflush=Whether every event should be immediately flushed to disk.
```

Comment:
might be worth stating that if the attribute is not defined, its value is derived from `synchronized`.